pkg/ccl/backupccl/testdata/backup-restore/descriptor-conflicts

```diff
@@ -69,56 +69,38 @@ SELECT count(1) FROM [SHOW TABLES FROM system] WHERE table_name LIKE 'crdb_inter
 query-sql
 SELECT max(id) FROM system.namespace WHERE name NOT LIKE 'crdb_internal_copy_%';
 ----
-112
+133
 
-# We should have remapped the 4 conflicting system tables to IDs higher than any
-# descriptor in the backup.
-query-sql
-SELECT id FROM system.namespace WHERE name LIKE 'crdb_internal_copy_%' ORDER BY id;
-----
-113
-114
-115
-116
-
-# Be paranoid and check that there is no descriptor with ID above the 4th remapped system table.
+# We should have left the 4 system tables where they were.
 query-sql
-SELECT count(*) FROM system.namespace WHERE id > 116;
+SELECT name, id FROM system.namespace WHERE id IN (104,105,106,107);
 ----
-0
+crdb_internal_copy_104 104
+crdb_internal_copy_105 105
+crdb_internal_copy_106 106
+crdb_internal_copy_107 107
 
-# The system table IDs should have been replaced by restored descriptors.
 query-sql
-SELECT name, id FROM system.namespace WHERE id IN (104,105,106,107);
+SELECT name, id FROM system.namespace WHERE name LIKE 'crdb_internal_copy_%';
```

Comment:
any reason to keep this statement around? Seems redundant with the previous one.

Reply:
I figured we want to test both directions on the namespace table since the conflicts bug created duplicate entries and all